### ISSUE TITLE
Supressing IF NOT EXISTS, since it is not a strict SQL92 phrase

### ISSUE THREAD

**Issue by kerusan** [vapor/sql-kit#67]
Code snippet
```
        try self.db.create(table: "planets")
            .ifNotExists()
            .column("id", type: .int, .primaryKey)
            .column("galaxyID", type: .int, .references("galaxies", "id"))
            .run().wait()

```

Generates SQL as

`CREATE TABLE IF NOT EXISTS "planets"("id" INTEGER PRIMARY KEY, "galaxyID" INTEGER REFERENCES "galaxies"
`

The phrase IF NOT EXISTS is not part of SQL 92, and is not recognized by databases like Frontbase. Some way of suppressing the .ifNotExists() function would be preferable when using a database that doesn’t support it.

**Comment by tanner0101** [comment]
I'd recommend the same thing here as #63

**Comment by kerusan** [comment]
+1

**Comment by tanner0101** [comment]
Fixed in #68
